Assets/HoloToolkit-Examples/Boundary/Scripts/BoundaryTest.cs

```diff
@@ -28,6 +28,17 @@ private void Start()
                     defaultMaterials[i].SetColor("_Color", highlightColor);
                 }
             }
+            else
+            {
+                Debug.LogFormat("Object " + name + " is outside established boundary. Position: {0}", gameObject.transform.position);
```

Comment:
Should this be a warning?

Reply:
It's more of an informative message in the test scene, showing which of the pre-existing cubes are inside/outside the boundary.

Reply:
```
Debug.LogFormat("Object {0} is outside established boundary. Position: {1}", name, gameObject.transform.position);
```